docs/docs/product-documentation.md

```diff
@@ -75,54 +74,58 @@ The strategic aim of the Swapchain team is to develop an HTLC-backed cryptocurre
+Despite the complexity of its underlying technology and the expected skill level of its users regarding cryptocurrencies, the design of the Swapchain interface is held in a minimalistic, unclogged structure. This is due to the fact that for ACCS input fields for private keys, public keys, addresses, fees, timelocks and bid and ask amounts and types are needed. To simply ask the user for these input is sufficient to be able to conduct an ACCS. Therefore, no further "fancy" functionalities are implemented in the prototype. The application is mobile-responsive to make trading comfortable with any device.
 
 Intuitivity:  
-Second, the Swapchain application is expected to provide an intuitive, easy-to-use interface. While related to the first aspect (simplicity), ease-of-use also includes the necessity of aspects such as obvious calls to action, for example if users needs to register for an account first or want to quickly browse existing swap offers. Moreover, users must be enabled to give input into the application without circumstances, which could for instance appear in the form of autofill patterns if they repeatedly want to set up the same swap offer. Moreover, barrier-free accessibility is also an important topic to be covered, however this functionality makes sense to be rolled out only when all other basic features have been implemented. Then, voice commands (of course excluding sensitive parts such as private keys) could prove viable support for impaired people.
+Second, the Swapchain application is expected to provide an intuitive, easy-to-use interface. While related to the first aspect (simplicity), ease-of-use also includes the necessity of aspects such as obvious call to action to start trading. Moreover, users must be enabled to give input into the application without circumstances, which could for instance appear in the form of autofill patterns if they repeatedly want to set up the same swap offer. Moreover, barrier-free accessibility is also an important topic to be covered, however this functionality makes sense to be rolled out only when all other basic features have been implemented. Voice commands (of course excluding sensitive parts such as private keys) could prove viable support for impaired people imaginably. 
 
-Adaptability:  
+Adaptability: 
+//den Teil verstehe ich nicht so recht, lass uns da noch mal drüber quatschen 
```

Comment:
Bezieht sich auf den zukünftigen zusätzlichen Benefit, also in welche Richtung man die Software noch weiter entwickeln bzw. was man Usern zusätzlich anbieten könnte. So wäre z.B. eine Idee, dem User in naher Zukunft Analysetools zur Verfügung zu stellen, mit denen er nachvollziehen kann, ob seine bisherigen Exchanges hinsichtlich Währungskursen etc. ökonomisch "sinnvoll" waren. Und für bessere spätere Anpassbarkeit der Software sollte diese natürlich Schnittstellen anbieten, nicht von Anfang an zu überladen sein usw.